node/src/main/scala/coop/rchain/node/TerminalMode.scala

```diff
@@ -0,0 +1,16 @@
+package coop.rchain.node
+
+trait TerminalMode[F[_]] {
+  def interactive: Option[Unit]
```

Comment:
`Option[Unit]` is isomorphic to `Boolean`, so we should be using `Boolean` here (use the simplest type possible)

Reply:
I agree with `Boolean`, not see the point of `Option[Unit]` here

Reply:
since it's isomorphic - why not use something that has a  `.map`?

Reply:
Because it's misleading. Also: why not just use an `if` instead, or convert the boolean to an option in the call site?